packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -281,122 +32,3 @@ QUnit.module('{{#with}} helper binding to view keyword', {
 QUnit.test('{{with}} helper can bind to keywords with \'as\'', function() {
   equal(view.$().text(), 'We have: this is from the view and this is from the context', 'should render');
 });
-
-testWithAs('ember-htmlbars: {{#with x as |y|}}', '{{#with person as |tom|}}{{title}}: {{tom.name}}{{/with}}');
-
-QUnit.module('Multiple Handlebars {{with foo as |bar|}} helpers', {
-  setup() {
-    Ember.lookup = lookup = { Ember: Ember };
-  },
-
-  teardown() {
-    runDestroy(view);
-    Ember.lookup = originalLookup;
-  }
-});
-
-QUnit.test('re-using the same variable with different #with blocks does not override each other', function() {
-  view = EmberView.create({
-    template: compile('Admin: {{#with admin as |person|}}{{person.name}}{{/with}} User: {{#with user as |person|}}{{person.name}}{{/with}}'),
-    context: {
-      admin: { name: 'Tom Dale' },
-      user: { name: 'Yehuda Katz' }
-    }
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'Admin: Tom Dale User: Yehuda Katz', 'should be properly scoped');
-});
```

Comment:
Direct port: re-using the same variable with different #with blocks does not override each other

:ok_hand: